radio/src/targets/taranis/hal.h

```diff
@@ -223,7 +223,7 @@
   // ROTARY_ENCODER_EXTI_LINE1 IRQ
   #if !defined(USE_EXTI9_5_IRQ)
     #define USE_EXTI9_5_IRQ
-    #define EXTI9_5_IRQ_Priority 5
+    #define EXTI9_5_IRQ_Priority 0
```

Comment:
Please use some `define` to make the intent clearer. Something like this up-front:
```
#define TELEMETRY_EXTI_PRIO 0
```

And then using this everywhere you change the prio:
```
  #define EXTI9_5_IRQ_Priority TELEMETRY_EXTI_PRIO
```

Reply:
done